firebase/src/firebase/java/org/mozilla/focus/utils/FirebaseWrapper.java

```diff
@@ -74,12 +74,12 @@
 
 
     // get Remote Config string
-    static String getRcString(@NonNull Context context, @NonNull String key) {
```

Comment:
You might need to change the one in firebase_no_op/.../FirebaseWrapper.java as well.